.github/workflows/main.yml

```diff
@@ -22,3 +22,45 @@ jobs:
         node:
           - lts/erbium
           - node
+  canary:
+    name: canary / ${{matrix.package}} / ${{matrix.node}} on ${{matrix.os}}
+    runs-on: ${{matrix.os}}
+    steps:
+      - name: checkout unified
+        uses: actions/checkout@v2
+      - name: setup node
+        uses: dcodeIO/setup-node-nvm@master
+        with:
+          node-version: ${{matrix.node}}
+      - name: setup unified
+        run: |
+          npm install -g npm
+          npm install
+          npm run build
+      - name: checkout ${{matrix.package}}
+        uses: actions/checkout@v2
+        with:
+          repository: ${{matrix.package}}
+          path: canary/${{matrix.package}}
+      - name: test ${{matrix.package}}
+        run: |
+          npm install
+          npx rimraf "node_modules/**/unified"
```

Comment:
unlike micromark, this is a single package, so the folder name can be hard coded